src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/terminal.chatAgentTools.contribution.ts

```diff
@@ -27,7 +31,22 @@ import { CreateAndRunTaskTool, CreateAndRunTaskToolData } from './tools/task/cre
 import { GetTaskOutputTool, GetTaskOutputToolData } from './tools/task/getTaskOutputTool.js';
 import { RunTaskTool, RunTaskToolData } from './tools/task/runTaskTool.js';
 
-// #region Workbench contributions
+class ShellIntegrationTimeoutMigrationContribution extends Disposable implements IWorkbenchContribution {
+	static readonly ID = 'terminal.shellIntegrationTimeoutMigration';
+
+	constructor(
+		@IConfigurationService private readonly _configurationService: IConfigurationService,
```

Comment:
Should not be `private readonly` or have `_` prefix as it's only used in the ctor.